src/cmd_argument.rs

```diff
@@ -21,12 +22,42 @@ where
     }
 }
 
+/// Arguments of type [`OsString`] are passed to the child process
+/// as arguments.
+///
+/// ```
+/// use cradle::*;
+///
+/// cmd_unit!("echo", std::env::var_os("PATH").unwrap());
+/// ```
+impl CmdArgument for OsString {
+    #[doc(hidden)]
+    fn prepare_config(self, config: &mut Config) {
+        config.arguments.push(self);
+    }
+}
+
+/// Arguments of type [`OsStr`] are passed to the child process
+/// as arguments.
+///
+/// ```
+/// use cradle::*;
+///
+/// cmd_unit!("echo", std::env::current_dir().unwrap().file_name().unwrap());
```

Comment:
Same as above, I feel there should be a better example than just using `echo`. But I'm also having a hard time coming up with one... documentation is hard.